### PR TITLE
Supports the new Intercom Messenger

### PR DESCRIPTION
**Pull request by eraad**
My user data was not being correctly reported after activating the new Intercom Messenger.

These changes were suggested by Intercom support:
- Merge custom data with main user data.
- Delete the widget code as it is no longer needed

Reference:
https://docs.intercom.com/configure-intercom-for-your-product-or-site/customize-intercom-to-be-about-your-users/send-custom-user-attributes-to-intercom

screen shot 2016-10-06 at 7 42 54 am

screen shot 2016-10-06 at 7 43 14 am

### CODE REVIEW

**Comment by eraad** [comment]
@kencochrane I did not send this PR from my company account since we added an extra parameter to cd_class.company_data to receive session data (https://github.com/datil/django-intercom/commit/2186398014037e39d75900701d9313a0eb96c914). I believe that would be a breaking change as it is implemented now.

File: intercom/templatetags/intercom.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def intercom_tag(context):
                     log.warning("%s couldn't be imported, there was an error during import. "
                                 "skipping. %s", custom_data_class, e)
 
-            custom_data = json.dumps(custom_data)
+            custom_data = custom_data
```
Comment (kencochrane) [inline intercom/templatetags/intercom.py]: we can probably just remove this, since it is now redundant.

Comment (eraad) [inline intercom/templatetags/intercom.py]: @kencochrane Great, thanks. Will do the changes and change the Javascript code too, I just had another interaction with Intercom support about updating the client side code.

Comment (kencochrane) [inline intercom/templatetags/intercom.py]: Cool, thank you.

**Comment by eraad** [comment]
@kencochrane Hi, just pushed the requested changes. Sorry for taking so long.

**Comment by kencochrane** [comment]
LGTM thank you.

**Pull request merged**